.github/workflows/release-state-manager.yml

```
@@ -0,0 +1,113 @@
+          name: state-manager-test-results
+          path: state-manager/pytest-report.xml
+          retention-days: 30
+
+  publish-image:
+    runs-on: ubuntu-latest
+    needs: test
+
+    permissions:
+      contents: read
+      packages: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)